dune-precice-howto/examples/dune-perpendicular-flap.cc

```diff
@@ -1,6 +1,3 @@
-// -*- tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 2 -*-
-// vi: set et ts=4 sw=2 sts=2:
```

Comment:
These instructions seem important. Are we sure we don't want them here anymore?

Reply:
Okay. In that case, these comments do not seem very relevant, and we can remove them.